### DISCUSSION TITLE
how to override sm

### DISCUSSION THREAD

**Discussion by im-pan** [unocss/unocss#2984] [Q&A] [answered]
Hello i have a problem
I want to add a mm: or override sm: to make his decision as follows:
=>  @media only screen and (hover: none) and (pointer: coarse) {}
What should I do?

**Comment by zyyv** [comment]
What's your expect css output?
Redefining breakpoints is easy, https://unocss.dev/config/theme#usage-in-rules. But I think you can still need to customize a variant.

**Comment by im-pan** [comment]
I want to achieve something like this
<img width="589" alt="image" src="https://github.com/unocss/unocss/assets/5241116/55afa8ba-231c-463d-8bf8-8c781b36c770">
so my unocss config
<img width="657" alt="image" src="https://github.com/unocss/unocss/assets/5241116/4f5844b3-41b3-4d6f-9376-a7eef4ea83bc">
and my jsx code 
<img width="601" alt="image" src="https://github.com/unocss/unocss/assets/5241116/27bbc9e2-1829-4e68-8843-8ab27cd82feb">

However, it's not working. I've looked through the documentation and haven't found a good solution. Did I miss something?

**Comment by zyyv** [comment, marked as answer]
[Playground](https://unocss.dev/play/?html=DwEwlgbgBAxgNgQwM5ILwCIAuBTJmC0OAHgQE7YjoB8AqgHYD2AwgMovAD04EVAUEA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbjlJtKgBccADa3j6%2BAjJy7gCUJGE%2B-FIMTKwcXNyx8eERUiJingnhVMJWtqhBAIwAdABM2TlwwshYQRoAFoxggQD0PdQg1VTtPRoNPgpxYQC6DQBuVsxWWDCBIUVejXBYViAVmjDUMGNFPiBWMMLt7ueX7ahQMYin4ZQwbFBYz1s5t1cPQT%2B9ygQ1swGEqHcGkOVBgAFoNNVylgiDB2lMfuEwItUCs2gABPbIYBWOAEWzcOAlSi4uDLZBwdztCBzAHbAioJ70xmQbCHKBBUSLGgxE6YhQvBTjRQNWYGSYGIA&css=Q&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)

**Comment by im-pan** [comment]
This is accurate and I greatly appreciate it. I suggest the documentation could offer additional examples. I've conducted numerous experiments and written similar code. I plan to emulate the provided examples and initiate crafting my own preset. Thank you once again ~

**Reply by zyyv** [reply]
You are welcome, but you should differentiate when to use `variant` and `theme`.
